Fix:

error: implicitly declaring library function 'tolower' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'toupper' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'time' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- mkpwd.c.orig	2012-08-02 10:22:13.000000000 -0500
+++ mkpwd.c	2024-01-08 00:21:04.000000000 -0600
@@ -25,6 +25,8 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <string.h>
+#include <ctype.h>
+#include <time.h>
 #include <errno.h>
 
 const char* PROGRAM	= "mkpwd";
